File: Day1/factorialCalculator.js
function getFactorial(num){
    if(num<0){
        return -1;
    }
    if(num ==0){
        return 1;
    }
    else{
        let fact =1;
        for(let i=1; i<= num; i++){
            fact = fact*i;
        }
        return fact;
    }
}

let num = parseInt(prompt("Enter the number:"));

let ans =getFactorial(num);
if(ans == -1){
        console.log("The Factorial of a Negative number is not defined");
    }
else{
    console.log("The Factorial is : " + ans );
    }

File: Day1/FactorialCalculator.java
import java.util.Scanner;
public class FactorialCalculator {
    public static int getFactorial(int num){
        if( num < 0){
            return -1;
        }
        if( num ==0){
            return 1;
        }
        int fact = 1;
        for(int i=1; i <= num; i++){
            fact *= i;
        }
        return fact;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int num = sc.nextInt();
        sc.close();

        int ans =getFactorial(num);
        if(ans == -1){
            System.out.println("The Factorial of a Negative number is not defined");
        }
        else{
            System.out.println("The Factorial is : " + ans );
        }
    }
}
